lib/init/npm-utils.js

```diff
@@ -119,12 +119,12 @@ function check(packages, opt) {
         throw error;
     }
 
-    if (opt.devDependencies && typeof fileJson.devDependencies === "object") {
-        deps = deps.concat(Object.keys(fileJson.devDependencies));
-    }
-    if (opt.dependencies && typeof fileJson.dependencies === "object") {
-        deps = deps.concat(Object.keys(fileJson.dependencies));
-    }
+    ["dependencies", "devDependencies"].forEach(key => {
```

Comment:
It might not matter, but it looks like this actually does change the behavior of the code because it switches the order in which it concats these arrays. Should we make it match the original code?

Reply:
It may not matter, IMHO.